cockroachcloud/export-logs.md

```diff
@@ -186,7 +186,7 @@ Perform the following steps to enable log export from your {{ site.data.products
                      },
                      {
                          log_name: "devops",
-                         channels: ["OPS", "HEALTH", "STORAGE"]
+                         channels: ["OPS", "HEALTH", "STORAGE"],
                          min_level: "WARNING"
                      },
              ]
```

Comment:
This entire snippet should have quoted keys as well as values

Reply:
Also quote the value `AWS_CLOUDWATCH` but not the value `true`.